visualize-admin/visualization-tool
# fix: Filters applying when adding a new chart

Fixes #1462

This PR fixes filters applying when adding a new chart.

## How to test
### PR
1. Go to [this link](https://visualization-tool-git-fix-adding-new-chart-when-fi-280eb5-ixt1.vercel.app/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/2&dataSource=Test).
2. Enable segmentation by Kanton.
3. Add a new Line chart.
4. ✅ See that the Kanton filter is correctly applied.

### TEST (also broken on PROD)
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/2&dataSource=Test).
3. Enable segmentation by Kanton.
4. Add a new Line chart.
5. ❌ See that the Kanton filter is not applied.
